vernac/comFixpoint.ml

```diff
@@ -224,10 +224,15 @@ let interp_recursive ~program_mode ~cofix fixl notations =
 
   (* Instantiate evars and check all are resolved *)
   let sigma = solve_unif_constraints_with_heuristics env_rec sigma in
-  let sigma = Evd.minimize_universes sigma in
-  (* XXX: We still have evars here in Program *)
-  let fixdefs = List.map (fun c -> Option.map EConstr.(to_constr ~abort_on_undefined_evars:false sigma) c) fixdefs in
-  let fixtypes = List.map EConstr.(to_constr sigma) fixtypes in
+  let sigma, (fixdefs, fixtypes) =
+    (* NB: we still have evars in program mode *)
+    Evarutil.finalize ~abort_on_undefined_evars:false
+      env sigma
+      (fun nf ->
+         let fixdefs = List.map (fun c -> Option.map nf c) fixdefs in
+         let fixtypes = List.map nf fixtypes in
+         fixdefs, fixtypes)
+  in
```

Comment:
Looks like color is failing in Program Fixpoint https://github.com/fblanqui/color/blob/2d02c602e9d8ef096c9a2a3bb66a3784172230bb/Util/Vector/VecUtil.v#L1758 
The `rec_sign` seems like the only possible source of leaks. Will try passing it through finalize.

Reply:
That worked.